docs/useRegisterMutationMiddleware.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: "useRegisterMutationMiddleware"
+---
+
+# `useRegisterMutationMiddleware`
+
+This hook allows to register a mutation middleware function.
```

Comment:
This isn't really helping if you don't know what a middleware function is...

Reply:
Same